extensions/grpc/deployment/src/test/java/io/quarkus/grpc/client/deadline/HelloService.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.grpc.client.deadline;
+
+import io.grpc.examples.helloworld.Greeter;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+import io.quarkus.grpc.GrpcService;
+import io.smallrye.mutiny.Uni;
+
+@GrpcService
+public class HelloService implements Greeter {
+
+    @Override
+    public Uni<HelloReply> sayHello(HelloRequest request) {
+        try {
+            Thread.sleep(400l);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        return Uni.createFrom().item(HelloReply.newBuilder().setMessage("OK").build());
```

Comment:
To avoid the sleep, what about adding:

OnItem().delayIt().by(Duration.ofMillis(400))

It also avoids the previous comment.

Reply:
Yay! +1